configure.ac

```diff
@@ -4688,6 +4688,12 @@ fi
 
 SDL_STATIC_LIBS="$EXTRA_LDFLAGS"
 
+dnl Calculate the location of the prefix, relative to the cmake folder
+eval pkg_prefix=$prefix
+eval pkg_cmakedir=$libdir/cmake/SDL2
+cmake_prefix_relpath="$(echo -n "$pkg_cmakedir" | sed -E "s#^$pkg_prefix##" - | sed -E "s#[A-Za-z0-9_ .-]+#..#g" -)"
+AC_SUBST([cmake_prefix_relpath])
```

Comment:
I could not use `realpath --relative-to=` because it expects folders/files to exist.

Also, I'm no shell wizard, so this code might not be compatible with other shells.
I only tested on my system bash 5.1.

Reply:
realpath isn't available on all supported platforms, so this is a good change.

Reply:
It looks like it fails on Macos:
```
sed: -: No such file or directory
sed: -: No such file or directory
```
I think we need to use a purpose built macro instead of the kludge I wrote.
e.g. [`ax_compute_relative_paths`](https://www.gnu.org/software/autoconf-archive/ax_compute_relative_paths.html)

But I don't know what's acceptable license or dependency wise.

Reply:
Just omit the '-', sed reads from standard input by default.